src/menu.cpp

```diff
@@ -1021,15 +1035,39 @@ class ControlsMenu final : public MenuBase
 
 const ControlsMenu::KeySettings ControlsMenu::c_key_settings[]=
 {
-	{ "Move Forward"	, SettingsKeys::key_forward	, KeyCode::W },
-	{ "Move Backward"	, SettingsKeys::key_backward	, KeyCode::S },
-	{ "Strafe Left"		, SettingsKeys::key_step_left	, KeyCode::A },
-	{ "Strafe Right"	, SettingsKeys::key_step_right	, KeyCode::D },
-	{ "Turn Left"		, SettingsKeys::key_turn_left	, KeyCode::Left },
-	{ "Turn Right"		, SettingsKeys::key_turn_right	, KeyCode::Right },
-	{ "Look Up"		, SettingsKeys::key_look_up	, KeyCode::Up },
-	{ "Look Down"		, SettingsKeys::key_look_down	, KeyCode::Down },
-	{ "Jump"		, SettingsKeys::key_jump	, KeyCode::Space },
+	/* essential controls */
```

Comment:
![controls_new](https://github.com/Panzerschrek/Chasm-Reverse/assets/2518812/0d058876-7542-4c0c-a3f6-6669b802ca51)
![controls_old](https://github.com/Panzerschrek/Chasm-Reverse/assets/2518812/2c2906a3-fa38-4c4a-911e-9a9abaca90fb)

Looks better, but still - not good enough.
I still suggest to add controls submenu with advanced controls - all added in this PR and (maybe) also controls for weapons selection.

Reply:
Adapted control menu style to look like the original version plus added weapon next/prev and added keypad scancodes